app/models/developer.rb

```diff
@@ -29,6 +29,7 @@ class Developer < ApplicationRecord
   has_one :role_level, dependent: :destroy, autosave: true
   has_one :role_type, dependent: :destroy, autosave: true
   has_one_attached :cover_image
+  has_many :developer_external_profiles, dependent: :destroy
```

Comment:
```suggestion
  has_many :external_profiles, dependent: :destroy, class_name: "Developers::ExternalProfile"
```

I think that's the correct syntax.